hadoop-ozone/tools/src/main/java/org/apache/hadoop/ozone/shell/tenant/RevokeUserAccessToTenantHandler.java

```diff
@@ -15,18 +15,18 @@
  *  See the License for the specific language governing permissions and
  *  limitations under the License.
  */
-package org.apache.hadoop.ozone.shell.s3;
+package org.apache.hadoop.ozone.shell.tenant;
 
 import org.apache.hadoop.ozone.client.OzoneClient;
 import org.apache.hadoop.ozone.shell.OzoneAddress;
 import picocli.CommandLine;
 
 /**
- * ozone s3 user modify.
+ * ozone tenant user revoke.
  */
-@CommandLine.Command(name = "modify",
-    description = "Modify a tenant user")
-public class TenantUserModifyHandler extends S3Handler {
+@CommandLine.Command(name = "revoke",
+    description = "Revoke user access to tenant")
+public class RevokeUserAccessToTenantHandler extends TenantHandler {
```

Comment:
this would be a tenant admin command. right ? invoker should have "ALL" access to the tenant volume.

Reply:
Yeah. Besides admins, I think one can also revoke him/herself from a tenant?

I'm not implementing `RevokeUserAccessToTenantHandler` in this jira. Just fixing some framework stuff that **remained** in the prototype code for now so it's easier for me to work on it later.